operate/webapp/src/main/java/io/camunda/operate/webapp/security/auth/BasicAuthenticationUserService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.security.auth;
+
+import io.camunda.authentication.ConditionalOnAuthenticationMethod;
+import io.camunda.authentication.entity.CamundaUser;
+import io.camunda.operate.webapp.rest.dto.UserDto;
+import io.camunda.operate.webapp.security.Permission;
+import io.camunda.security.entity.AuthenticationMethod;
+import java.util.List;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnAuthenticationMethod(AuthenticationMethod.BASIC)
+@Profile("consolidated-auth")
+public class BasicAuthenticationUserService extends AuthUserService {
+
+  @Override
+  public UserDto createUserDtoFrom(final UsernamePasswordAuthenticationToken authentication) {
+    final CamundaUser user = (CamundaUser) authentication.getPrincipal();
+    return new UserDto()
+        .setUserId(user.getUserId())
+        .setDisplayName(user.getDisplayName())
+        .setCanLogout(true)
+        .setPermissions(List.of(Permission.READ, Permission.WRITE));
```

Comment:
❓ I guess here we actually need to resolve the actual permissions? Can we actually cover this with https://github.com/camunda/camunda/issues/27885 ? Will we actually keep that service going forward or have something central from the authentication module?

Reply:
I think the context of [this](https://camunda.slack.com/archives/C06UYJMMETZ/p1737026427216549) thread may help here but the way I see it is the change to support this is larger, for example we need to use the new authorizations to correctly derive if `READ` and/or `WRITE` is present (using some form of mapping logic between new and old). 

I would agree with your suggestion though that we could consider it a part of  https://github.com/camunda/camunda/issues/27885 and make the issue a little broader to not only consider app access but also authorizations

Reply:
Actually READ/Write will vanish and it will only be ACCESS according to https://docs.google.com/document/d/1ikxw9ARCv-MBwASnj7WhW5uHCndCRk2B35xi-cB2N0M/edit?tab=t.0#heading=h.c0kx9z3moo6t

Reply:
Eventually yes, but there are some stage between now and then, specifically I think the Operate UI changes (maybe the v2 api switch?)